README.md

```diff
@@ -41,23 +41,28 @@ The list of current supported providers:
 - [Gate](https://www.gate.io/)
 - [Huobi](https://www.huobi.com/en-us/)
 - [Kraken](https://www.kraken.com/en-us/)
+- [Kujira](https://github.com/ojo-network/kujira-api)
 - [Mexc](https://www.mexc.com/)
 - [Okx](https://www.okx.com/)
-- [Osmosis](https://app.osmosis.zone/)
 - [OsmosisV2](https://github.com/ojo-network/osmosis-api)
+- [Polygon](https://api.polygon.io)
 <!-- markdown-link-check-enable -->
 
 ## Usage
 
-The `price-feeder` tool runs off of a single configuration file. This configuration
+The `price-feeder` tool runs off of two configuration files. The provider-config
```

Comment:
Could be more than two files now that the provider-config is a directory

Reply:
Good call, README has been updated with some clarification.